conn_linux_test.go

```diff
@@ -269,9 +269,51 @@ func TestLinuxValidate(t *testing.T) {
 	}
 }
 
-func testLinuxConn(t *testing.T) (*conn, *testSocket) {
+func TestLinuxConnConfig(t *testing.T) {
+
```

Comment:
I know this is silly, but I'm not a fan of whitespace at the start or end of function bodies.  Mind removing this one, lines 298, 305, and 310?  Sorry.  Just a personal preference.